cornac/serving/app.py

```diff
@@ -150,6 +157,46 @@ def add_feedback():
 
     return jsonify(data), 200
 
+@app.route("/evaluate", methods=["GET"])
```

Comment:
We should make it a POST request with body including an array of metric constructors (e.g., `{"metrics": ["RMSE()", "NDGC(k=20)"]}` and return the result accordingly.

Reply:
Ideally, we want it to be flexible so that users could specify metrics of their interest. Let’s try to do something similar to the models.